package.json

```diff
@@ -90,14 +96,11 @@
   ],
   "scripts": {
     "no-prepublish": "npm run build",
-    "no-build": "webpack",
+    "build": "webpack --mode production",
```

Comment:
Could move the production flag into the config as `mode: 'production'`.

Reply:
Moved, thx.